components/feature/addons/src/main/res/values/strings.xml

```diff
@@ -107,5 +109,7 @@
     <string name="mozac_feature_addons_updater_notification_channel">Addon updates</string>
     <!-- This is the caption for the addon installation progress overlay -->
     <string name="mozac_add_on_install_progress_caption">Downloading and verifying add-on&#8230;</string>
+    <!-- This is the caption for not yet supported screen caption -->
+    <string name="mozac_add_on_not_yet_supported_caption">Firefox add-on technology is modernizing. These add-ons use frameworks that are not compatible with Firefox 75 &amp; beyond.</string>
```

Comment:
nit: let's fix up the naming here. `mozac_feature_addons_....` instead of `mozac_add_on...`